x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/asset_criticality/constants.ts

```diff
@@ -16,6 +16,9 @@ const assetCriticalityMapping = {
   required: false,
 };
 
+// Upgrade this value to force a mappings update on the next Kibana startup
+export const ASSET_CRITICALITY_MAPPINGS_VERSIONS = 2;
```

Comment:
I wonder if we could have a unit test which asserts if the mapping has changed and lets people know to change the version, maybe just a snapshot would work

Reply:
I added two dumb tests, one for asset and one for risk score, to remind engineers to bump the version.